extensions/amp-image-slider/0.1/amp-image-slider.js

```diff
@@ -112,22 +112,21 @@ export class AmpImageSlider extends AMP.BaseElement {
   buildCallback() {
     const children = this.getRealChildren();
 
-    for (let i = 0; i < children.length; i++) {
-      const child = children[i];
-      if (child.tagName.toLowerCase() === 'amp-img') {
+    for (const child of children) {
```

Comment:
Re: chat thread earlier on for..of
```suggestion
    for (let i = 0; i < children.length; i++) {
      const child = children[i];
```

Reply:
Turns out `for ... of` is roughly the same performance as a forward loop. See updated thread.